### PR TITLE
Use org.embulk.spi.json.JsonValue instead of org.msgpack.value.Value

### PR DESCRIPTION
**Pull request by dmikurube**

### CODE REVIEW

File: src/main/java/org/embulk/input/config/ConfigInputPlugin.java [hunk 1]
```diff
@@ -181,8 +182,10 @@ public void jsonColumn(Column column) {
                                 pageBuilder.setNull(column);
                             } else {
                                 try {
-                                    pageBuilder.setJson(column, jsonParser.parse(value.toString()));
-                                } catch (JsonParseException ex) {
+                                    pageBuilder.setJson(column, jsonParserBuilder.build(value.toString()).readJsonValue());
+                                } catch (final JsonParseException ex) {
+                                    throw new DataException(ex);
```
Comment (hiroyuki-sato) [inline src/main/java/org/embulk/input/config/ConfigInputPlugin.java:187]: Nit: Do we need `TODO` comment like `util-csv`?
https://github.com/embulk/embulk-parser-csv/pull/6/files#diff-cf388e19f294b56513c5fdab2f16a64ef08dff3d22365a969a84c9652a3cfdb1R425

**Review by hiroyuki-sato — APPROVED** [review]
LGTM👍 
(Left one minor comment)

File: src/main/java/org/embulk/input/config/ConfigInputPlugin.java (hunk 1 above)
Comment (dmikurube) [reply src/main/java/org/embulk/input/config/ConfigInputPlugin.java:187]: Ah, for this case, I don' think we need it. `embulk-input-config` should receive everything in the configuration.

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**